bench/dune

```diff
@@ -6,7 +6,12 @@
 (executable
  (name db_bench)
  (modules db_bench)
- (libraries fmt index.unix common lmdb bigstring rresult logs logs.fmt))
+ (libraries fmt index.unix common bigstring logs logs.fmt cmdliner))
+
+(executable
```

Comment:
It's not clear to me why you would want these to be two separate binaries; did you have a particular reason for doing it this way?

Reply:
I figured since I split the two benchmarks, might as well split the binary too. I don't have particularly strong opinions on this.